src/uv-common.c

```diff
@@ -310,10 +310,6 @@ int uv_udp_init_ex(uv_loop_t* loop, uv_udp_t* handle, unsigned flags) {
 
   rc = uv__udp_init_ex(loop, handle, flags, domain);
```

Comment:
```suggestion
  return uv__udp_init_ex(loop, handle, flags, domain);
```
And drop the `int rc;` a few lines up.